conf.default.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think json file is here to help give the default config to app without setup the environment variables. 
If we don't need env.dist to deploy the app on Heroku, I think we can delete it. I will update the README.md for the changes.